pkg/reconciler/v1alpha1/route/resources/service.go

```diff
@@ -38,9 +48,36 @@ func MakeK8sService(route *v1alpha1.Route) *corev1.Service {
 				*kmeta.NewControllerRef(route),
 			},
 		},
-		Spec: corev1.ServiceSpec{
+		Spec: *svcSpec,
+	}, nil
+}
+
+func makeServiceSpec(ingress *netv1alpha1.ClusterIngress) (*corev1.ServiceSpec, error) {
+	ingressStatus := ingress.Status
+	if ingressStatus.LoadBalancer == nil || len(ingressStatus.LoadBalancer.Ingress) == 0 {
+		return nil, errLoadBalancerNotFound
+	}
+	// TODO(lichuqiang): what to do when we have multiple ingresses here?
```

Comment:
Feels like something we should bias towards erroring on instead of silently ignoring?